.github/workflows/new_app_test.yml

```diff
@@ -0,0 +1,108 @@
+name: "Generate new app"
```

Comment:
Why do we need `test.yml` at all (I mean the other file)? I think this workflow could just replace it. So `new_app_test.yml` (which is not a great name 😆 ) would be `test.yml`, no?